### PR TITLE
DOC: Add examples to Series operators (#24589)

### PR DESCRIPTION
**Pull request by sullivanbt**
Further work on #24589.  PR #25524 added examples for many operations but not `pandas.Series.eq`, `pandas.Series.ne`, `pandas.Series.gt`, `pandas.Series.ge`, `pandas.series.le`, and `pandas.series.lt`.  This adds examples for those.

`pandas.Series.divmod` is still missing an example as discussed in #25524.

### CODE REVIEW

**Comment by datapythonista** [comment]
Thanks for working on this @sullivanbt 

What do you think about having in a variable the creation of the `Series`, which is repeated in every example, and then do something like:
```python
_ne_example_SERIES = _common_examples_SERIES + """
>>> a.ne(b, fill_value=0)
...
```
This would save a decent amount of repeated code.

**Comment by sullivanbt** [comment]
Thanks for the suggestion @datapythonista - I updated the branch with 926a3cf to include that.

**Comment by datapythonista** [comment]
Cool. May be it would make sense to rename `_common_examples_SERIES` to `_common_examples_algebra_SERIES` to be more explicit.

Did you render one of the pages to html? Would be good to render at least one of each (algebra/comparison), to make sure there are not too many or two few blank lines... If you can post couple of screenshots even better. `scripts/validate_docstrings pandas.Series.mul` can also be helpful to identify possible errors.

Good job, nice improvement.

**Comment by sullivanbt** [comment]
Completely agree that `_common_examples_algebra_SERIES` is a more useful name - updated in ca5111b.  

I've attached two screenshots - one from the algebra examples one from the comparison.  I believe this is how they should be rendering (same as the existing examples) but let me know if not.

Thanks again for your feedback.

image
image

**Review by datapythonista — APPROVED** [review]
Really cool @sullivanbt, looks perfect.

I'll let another maintainer merge, so we have couple of eyes on this, but this should be ready. Thanks!

**Comment by TomAugspurger** [comment]
Thanks @sullivanbt!

**Pull request merged**
